api/blob/azure_client.py

```diff
@@ -82,3 +90,20 @@ async def get_new_thumbnail(self, data_type: str, filepath: str) -> bytes:
         content = await self.get_blob_content(iq_path, 8000, fftSize * 512)
         image = get_spectrogram_image(content, data_type, fftSize)
         return image
+
+    async def get_medatada_files(self):
+        container_client = self.get_container_client()
+        # files that enf with .sigmf-meta
+        async for blob in container_client.list_blobs():
+            if blob.name.endswith(".sigmf-meta"):
+                blob_client = self.get_blob_client(blob.name)
+                blob = await blob_client.download_blob()
+                content = await blob.readall()
+                yield str(blob.name), Metadata.parse_raw(content)
+        return
+
+    async def get_metadata_file(self, filepath: str):
```

Comment:
is there a plan for when this would be used?

Reply:
This is used already by the sync method that was introduced as part of this pr.

Reply:
Sorry misread the name you were talking about getting a single file.yeah I was using ti before but not anymore but would make sense so strandarize and make it onwe method will group thodse thx

Reply:
Fixed